### PR TITLE
[Docs] Update issue templates

### PR DESCRIPTION
**Pull request by stichbury**
## Description
Maybe not exactly docs, more like a comms issue, to make it simpler to report bugs, request features and ask questions.

I've addressed #1098 from our internal issues queue to edit the issues templates and simplify them a little.

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/general-question.yml [hunk 1]
```diff
@@ -27,27 +24,12 @@ body:
...
       label: Which package?
       options:
         - vizro
         - vizro-ai
-  - type: input
-    attributes:
-      label: Package version
-  - type: input
-    attributes:
-      label: Python version
```
Comment (huong-li-nguyen) [inline .github/ISSUE_TEMPLATE/general-question.yml:47]: Do we ask for the package and python elsewhere? I think this might be useful to keep?

**Review by huong-li-nguyen — APPROVED** [review]
LGTM 👍 Have a minor comment, but ultimately don't mind if you remove it or not. But we do ask for the python and package version normally when we try to debug :)

**Comment by stichbury** [comment]
> LGTM 👍 Have a minor comment, but ultimately don't mind if you remove it or not. But we do ask for the python and package version normally when we try to debug :)

I feel like we ask too much, but I do understand it's useful to have this information (though we haven't made it a requirement in the past). How about I include it as something they should tell us, so it's still a suggested thing to include, but not in a separate field? I'll make the commit, see what you think @huong-li-nguyen

File: .github/ISSUE_TEMPLATE/bug-report.yml
Comment (antonymilne) [inline .github/ISSUE_TEMPLATE/bug-report.yml:1]: I would completely get rid of the following questions too:
* Expected behavior 
* Python version
* OS
* Output

Comment (antonymilne) [reply .github/ISSUE_TEMPLATE/bug-report.yml:1]: And let's move "Which package" and "Version" to the top of the form also I think.

File: .github/ISSUE_TEMPLATE/general-question.yml [hunk 2]
```diff
@@ -25,29 +22,14 @@ body:
...
       label: Code/Examples
       description: >-
-        Please post any supporting code around your question here.
-
-  - type: textarea
-    attributes:
-      label: Other information
-      description: >-
-        Please provide any attempts on answering the question, or any related information you already found here.
+        Please post any supporting code around your question here. Don't forget to include the Python version and package version, if it's relevant.
```
Comment (antonymilne) [inline .github/ISSUE_TEMPLATE/general-question.yml:25]:
```suggestion
        Please post any supporting code around your question here. 
```

Personally I agree with your original removal of this 🙂

File: .github/ISSUE_TEMPLATE/general-question.yml (hunk 1 above)
Comment (antonymilne) [reply .github/ISSUE_TEMPLATE/general-question.yml:47]: I don't think it's relevant for a general question, just for bug reports (and even then we don't really care about Python version).

**Review by antonymilne — APPROVED** [review]
Thanks for doing this, it looks a lot better! I suggest we go even more drastic with this - see comments.

Re: package and version - I think this is only relevant on the bug report so would only include it there. Python version and OS have never been relevant AFAIK so not worth mentioning at all - we can always follow up asking someone what those are if we suspect that might be the problem.

**Review by maxschulz-COL — APPROVED** [review]
LGTM! I would wait merging this until MOnday for when we have the py.cafe tile - then I would actively refer to py.cafe, and suggest people post there code problem via py.cafe if that is relevant

Then we can just click on the link and see the issue!

**Comment by maxschulz-COL** [comment]
> LGTM! I would wait merging this until MOnday for when we have the py.cafe tile - then I would actively refer to py.cafe, and suggest people post there code problem via py.cafe if that is relevant
> 
> Then we can just click on the link and see the issue!

@stichbury We now have the py.cafe tile - so maybe we can include that?

File: .github/ISSUE_TEMPLATE/bug-report.yml
Comment (stichbury) [reply .github/ISSUE_TEMPLATE/bug-report.yml:1]: Done!

**Comment by stichbury** [comment]
@maxschulz-COL I've added in a link to py.cafe in a couple of places. Please see what you think!

**Comment by maxschulz-COL** [comment]
> @maxschulz-COL I've added in a link to py.cafe in a couple of places. Please see what you think!

Yes I think it's great, let's go!

**Pull request merged**